src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -108,19 +104,7 @@ class AuthScreens extends React.Component {
         });
 
         // Fetch some data we need on initialization
-        NameValuePair.get(CONST.NVP.PRIORITY_MODE, ONYXKEYS.NVP_PRIORITY_MODE, 'default');
-        NameValuePair.get(CONST.NVP.IS_FIRST_TIME_NEW_EXPENSIFY_USER, ONYXKEYS.NVP_IS_FIRST_TIME_NEW_EXPENSIFY_USER, true);
-        App.getLocale();
-        PersonalDetails.fetchPersonalDetails();
-        User.getUserDetails();
-        User.getBetas();
-        User.getDomainInfo();
-        PersonalDetails.fetchLocalCurrency();
-        Report.fetchAllReports(true, true);
-        GeoLocation.fetchCountryCodeByRequestIP();
-        UnreadIndicatorUpdater.listenForReportChanges();
```

Comment:
I don't think we would want to have this happen more than once.

Reply:
Makes sense! Thanks!